pym/gentoolkit/eclean/search.py

```diff
@@ -332,6 +332,18 @@ def _non_destructive(
             deprecated.update(_deprecated)
         return pkgs, deprecated
 
+    def git_check(self, distdir):
+        """Checks $DISTDIR/git3-src for checkouts which are not in the vardb"""
+        expected_dirs = [
+            self.vardb._aux_env_search(i, ["EGIT_DIR"])["EGIT_DIR"]
+            for i in set(self.vardb.cpv_all())
+            if "live" in self.vardb.aux_get(i, ["PROPERTIES"])
+        ]
+        git_src = os.path.join(distdir, "git3-src")
+        gitdir = [os.path.join(git_src, i) for i in os.listdir(git_src)]
+        deprecated_git = [i for i in gitdir if i not in expected_dirs]
```

Comment:
Do you mean obsolete instead of deprecated?

Reply:
Yes I'll change that